### PR TITLE
Refresh designs

### PR DESCRIPTION
**Pull request by shrayasr**
## Description

Taking a 'card' approach instead of a 'table' approach. And keeping them as lean as possible 

Also including [feather icons](https://feathericons.com/) for jazzzz

### Screens

- [x] event_add.html
- [x] event_delete.html
- [x] event_edit.html
- [x] events.html
- [x] meetup_add.html
- [x] meetup_add_owner.html
- [x] meetup_delete.html
- [x] meetup_edit.html
- [x] meetup_owner_index.html
- [x] meetups.html
- [x] speaker_add.html
- [x] speakers.html
- [x] user_register.html
- [x] login

### Others

- ~~More contextual breadcrumbs -- instead of `Meetup / Events / Add` use `Chennaipy / Events / Add`~~
  - This doesn't make sense since clicking on `Chennaipy` in the above breadcrumb would take them to the list of ALL meetups, not just chennaipy's meetup. Not proceeding with this. 
- [x] Better UX
  - Introduce the name of the meetup where ever it makes sense. So that context is retained

### `meetups.html`

image

### `meetup_add.html`

image

### `meetup_edit.html`

image

### `meetup_delete.html`

image

### `meetup_owner_index.html`

image

### `meetup_add_owner.html`

image

### `events.html`

image

### `event_add.html`

image

### `event_edit.html`

image

### `event_delete.html`

image

### `login.html`

image

### `registeration.html`

image

### CODE REVIEW

File: guby_backend/templates/app/meetups.html [hunk 1]
```diff
@@ -28,45 +28,47 @@
...
+                <div class="card-body">
+                  <h3 class="card-title">{{ meetup.name }}</h3>
+                  <a class="btn btn-primary btn-sm" href="/app/meetups/{{ meetup.id }}/events">Manage events</a>
+                  <div class="btn-group">
+                    <a class="btn btn-sm btn-secondary" href="/app/meetups/{{ meetup.id }}/edit">Edit</a>
+                    <button type="button" class="btn btn-secondary dropdown-toggle dropdown-toggle-split" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                      <span class="sr-only">Toggle Dropdown</span>
+                    </button>
+                    <div class="dropdown-menu">
+                      <a class="dropdown-item" href="/app/meetups/{{ meetup.id }}/delete">Delete</a>
```
Comment (raghothams) [inline guby_backend/templates/app/meetups.html:59]: You need to add link to meetup edit page

Comment (shrayasr) [reply guby_backend/templates/app/meetups.html:59]: That's there [here](https://github.com/unnati-xyz/guby/pull/9/files/f5f543465a30d5d827fe86a5680b6737541e45bc#diff-213d8ce67c316255b1c07c27befb4348R54) 

Its a split dropdown with the primary action being 'edit' and 'delete' and 'manage owners' are secondary options

image

Comment (raghothams) [reply guby_backend/templates/app/meetups.html:59]: Oh right! To me it looked like I always need to select from dropdown :)

Comment (shrayasr) [reply guby_backend/templates/app/meetups.html:59]: It is confusing, yes. Let us ship this and check back if folks are confis

**Pull request merged**